core/client/fs/src/main/java/alluxio/client/file/cache/CacheEvictor.java

```diff
@@ -23,7 +23,7 @@
    */
   static CacheEvictor create() {
     // return corresponding CacheEvictor impl
-    return null;
+    return new LRUCacheEvictor();
```

Comment:
I would suggest to add user-facing propertyKey like `alluxio.user.client.cache.evictor` and allow to select different evictors (thought currently we have only one). This will make it easier to switch cache replacement policy without recompiling the src code

Reply:
Added.